src/components/hud/TileSummary.tsx

```diff
@@ -3,13 +3,14 @@ import { ProjectsDisplay } from './ProjectsDisplay'
 import { Tile } from '../../game_logic'
 import { dispatcher } from '../reducers'
 import CloseIcon from '@mui/icons-material/Close'
+import { tileCatalog } from '../../data/tile-catalog'
 
 export function TileSummary (props: {
-  tile?: Tile,
+  tile: Tile | null,
```

Comment:
It's nullable here because it's like that in the UIState interface.
It's like that in the UIState interface because `?` means explicitly `undefined` *or missing*. Helps protect accidentally forgetting to specify it.